docs/source/benchmark/submission.md

```diff
@@ -0,0 +1,97 @@
+# Submission
+
+To participate in the ManiSkill2 challenge, please register on the [challenge website](https://sapien.ucsd.edu/challenges/maniskill/2022/). After registering an account, [create/join a team](https://sapien.ucsd.edu/challenges/maniskill/challenges/ms2022/team). After creating/joining a team, you will be allowed to create submissions.
```

Comment:
these links are outdated. should be https://sapien.ucsd.edu/challenges/maniskill/ and https://sapien.ucsd.edu/challenges/maniskill/challenges/ms2/team